package.json

```diff
@@ -162,16 +162,16 @@
     "@visx/responsive": "^2.10.0",
     "@visx/shape": "^2.11.1",
     "@walletconnect/ethereum-provider": "^1.8.0",
-    "@web3-react/coinbase-wallet": "^8.0.34-beta.0",
-    "@web3-react/core": "^8.0.35-beta.0",
-    "@web3-react/eip1193": "^8.0.26-beta.0",
-    "@web3-react/empty": "^8.0.20-beta.0",
-    "@web3-react/gnosis-safe": "^8.0.6-beta.0",
-    "@web3-react/metamask": "^8.0.28-beta.0",
-    "@web3-react/network": "^8.0.27-beta.0",
-    "@web3-react/types": "^8.0.20-beta.0",
-    "@web3-react/url": "^8.0.25-beta.0",
-    "@web3-react/walletconnect": "^8.0.35-beta.0",
+    "@web3-react/coinbase-wallet": "8.0.34-beta.0",
```

Comment:
Yarn will not respect `^` versioning due to the suffix.
To avoid the assumption that we can upgrade these without updating the `package.json`, I've explicitly pinned the versions.